### DISCUSSION TITLE
Footsteps on Physical Materials

### DISCUSSION THREAD

**Discussion by corbin-poteet** [Sixze/ALS-Refactored#34] [Q&A] [answered]
Has anyone else experienced problems with the footstep system not working correctly in their project? From what I can tell, the trace that it does in the footstep anim notify doesn't return the physical material. The only time it works for me in on the right foot when jumping. I have all the physical material and surface material information set up correctly.

**Comment by Sixze** [comment, marked as answer]
Fixed in 8dc0301036943f6775e8a04532df503d5faac2fe.
